### PR TITLE
feat: auto pick non-breaking changes

### PR DESCRIPTION
**Pull request by tomjohnburton**
This will allow developers to add the `non-breaking` label to their PR which will automatically add the resulting merged commit to the latest release branch and assign @martin-chainflip as a reviewer

### CODE REVIEW

File: .github/workflows/ci-cherry-pick.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          git push origin ${{ env.BRANCH }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.CF_BACKEND_GITHUB_TOKEN }}
+          commit-message: Cherry-picked from ${{ env.COMMIT_SHA }}
+          title: "Cherry-Pick ${{ env.COMMIT_SHA }} to ${{ env.RELEASE_BRANCH }}"
+          branch: ${{ env.BRANCH }}
+          base: '${{ env.RELEASE_BRANCH }}' # Update with your current release branch
```
Comment (kylezs) [inline .github/workflows/ci-cherry-pick.yml:50]: What's the comment about?

**Review by dandanlen — COMMENTED** [review]
Can we make it a label instead of having to edit the commit message? Having the information in the commit message seems a bit unnecessary... It will appear in (by default) in changelogs etc.

File: .github/workflows/ci-cherry-pick.yml (hunk 1 above)
Comment (tomjohnburton) [reply .github/workflows/ci-cherry-pick.yml:55]: No reason, deleting

**Comment by tomjohnburton** [comment]
Done

> Can we make it a label instead of having to edit the commit message? Having the information in the commit message seems a bit unnecessary... It will appear in (by default) in changelogs etc.

Updated as requests

**Review by AlastairHolmes — APPROVED** [review]

**Pull request merged**